components/basecomponents/Exercise/ExerciseOrchestrator.tsx

```diff
@@ -40,18 +40,10 @@ interface ExerciseOrchestratorProps {
 // warning: language switching triggers change for all props
 export const ExerciseOrchestrator = ({ categories }: ExerciseOrchestratorProps) => {
   const lang = useLanguage();
-  const init = useExerciseStore((state) => state.init);
-  const setLang = useExerciseStore((state) => state.setLang);
-  const setCategories = useExerciseStore((state) => state.setCategories);
-  const getAllCategories = useExerciseStore((state) => state.getAllCategories);
+  const { init, setLang, setCategories, getAllCategories, status, exercise, start, restart, home, counter, size } = useExerciseStore(
```

Comment:
Toto není dobrý nápad. Jelikož změna jakékoliv položky bude prezentována jako změna všech položek, viz. manuál Zustand. Vím že to vypadá obludně ,ale takhle se s tím Zustandem pracuje.

Reply:
Ok, o Zustandu toho víš než já.